gef-ui/templates/partials/facility-guarantee.njk

```diff
@@ -154,25 +154,25 @@
           },
           items: [
             {
-              value: "in advance",
+              value: "In advance",
```

Comment:
not a deal-breaker at all, (not sure how this component works) but might it be easier making this array in JS and passing it as a variable to items: ? Can use constants then (if that is what it is using)

Reply:
Perhaps, pros and cons to this approach IMO - i'm just updating the values here though think should be done later